.github/ISSUE_TEMPLATE/esm-report.md

```diff
@@ -7,16 +7,21 @@ assignees: ''
 
 ---
 
+The following information is required if you are reporting an issue or a bug. Before creating an issue, we recommend comparing your configuration to the sample applications here: https://github.com/Esri/jsapi-resources/tree/master/esm-samples. 
+
 **@arcgis/core version**
 
 <!--
-  Which version of `@arcgis/core`, for example `@arcgis/core@4.21.2`? Run `npm ls @arcgis/core` in a terminal window.
+  Which version of `@arcgis/core`, for example `@arcgis/core@4.24.7`? Run `npm ls @arcgis/core` in a terminal window.
   Questions related to `next` releases of `@arcgis/core` need to be addressed on the https://github.com/Esri/feedback-js-api-next repo.
 -->
 
+**Github repository link** 
```

Comment:
Github -> GitHub

Reply:
Or maybe change this header to something like "Repro link"

Reply:
Updated. I like Repro, but it is really close in spelling to Repo. I figure we can give it a try.